people/ioana-chiorean

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: QA Team Lead, Open Source Evanghelist, Open Designer Enthusiast 
```

Comment:
Thanks @Volker-E -trying this again https://github.com/opensourcedesign/organization/pull/25